src/config/__tests__/__fixtures__/config.invalid.yaml

```diff
@@ -0,0 +1,6 @@
+---
+{
```

Comment:
I guess this is double invalid, its mising the lint _and_ bad syntax. Is that intention?

Reply:
Right. I split it in separate test cases. I've also added validation for empty config and missing command.